Formula/tensorflow.rb

```diff
@@ -0,0 +1,52 @@
+class Tensorflow < Formula
+  desc "Machine Intelligence Library"
+  homepage "https://tensorflow.org"
+  url "https://github.com/tensorflow/tensorflow.git",
+    :tag => "v0.7.1",
+    :revision => "028d0b46004c921acd48fdd0ec18128d79e18bf4",
+    :shallow => false
+
+  env :std
+
+  depends_on "bazel"
+  depends_on "swig" => :build
+
+  resource "wheel" do
+    url "https://pypi.python.org/packages/source/w/wheel/wheel-0.29.0.tar.gz"
+    sha256 "1ebb8ad7e26b448e9caa4773d2357849bf80ff9e313964bcaf79cbf0201a1648"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  def install
+    resource("wheel").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    resource("six").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    ENV["PYTHON_BIN_PATH"] = which "python"
```

Comment:
No, it's not. `./configure` requires user input for the `PYTHON_BIN_PATH` (see [these lines](https://github.com/tensorflow/tensorflow/blob/v0.7.1/configure#L4-L23)).